app/src/main/java/wilsonoh/sagyo/ui/TextFormatter.java

```diff
@@ -0,0 +1,72 @@
+package wilsonoh.sagyo.ui;
+
+import static wilsonoh.sagyo.ui.ColorCodes.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+import wilsonoh.sagyo.tasks.Task;
+
+/**
+ * Helper class for printing out formatted text
+ */
+public class TextFormatter {
+
+    public static final int DEFAULT_MAX_WIDTH = 50;
+    private int numIndent;
+    private int maxWidth;
+
+    public TextFormatter(int indentation, int maxWidth) {
+        this.numIndent = indentation;
+        this.maxWidth = maxWidth;
+    }
+
+    public TextFormatter(int indentation) {
+        this.numIndent = indentation;
+        this.maxWidth = DEFAULT_MAX_WIDTH;
+    }
+
+    /**
+     * Clear screen method which only works on unix systems for now
+     */
+    private void clearScreen() {
+        System.out.print("\033\143");
+    }
+
+    public static String[] getFormattedTaskStrings(ArrayList<Task> tasks) {
+        return IntStream.rangeClosed(1, tasks.size())
+            .mapToObj(idx -> String.format("%d: %s", idx, tasks.get(idx - 1)))
+            .toArray(String[] ::new);
+    }
+
+    /**
+     * Prints lines of text and prints them in a nice little box
+     *
+     * Partially adapted from https://stackoverflow.com/a/58780542 with
+     * a little python to java translation (java string formatting isn't the best...)
+     *
+     * @param lines Variable argument. Takes in any number of lines to be printed in a box
```

Comment:
I like how you described the purpose of the functions and also gave references to where you adapted the code from.